tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "noEmit": true,
     "jsx": "react",
     "allowJs": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "downlevelIteration": true
```

Comment:
I added it because of the [code below](https://github.com/yorkie-team/yorkie-codepair/pull/45/files#diff-75a5cbb43d1dd466fc40f917296f59b28236ee256bbd1301fb3579b123f3bdeeR132-R134
)
We should turn on this option when we want to iterate over an array or string type on Typescript
```js
for (const shape of shapes) {
    this.draw(shape, canvas);
}
```

If we use this option, the bundle size increases, but I used it for readability.🤔

https://mariusschulz.com/blog/downlevel-iteration-for-es3-es5-in-typescript
In conclusion, the bundler file grows to serve the emoji.